src/bindgen/cargo/cargo_expand.rs

```diff
@@ -24,6 +24,14 @@ pub enum Error {
     Compile(String),
 }
 
+/// Which Cargo profile (group) to use when expanding macros.
+pub enum Profile {
```

Comment:
Any reason to have this enum and the config.rs enum duplicated?

Reply:
I wasn't really sure how to do the dependencies. If the canonical version is in `cargo_expand`, then `cargo_expand` is exporting public types now; if the canonical version is in `config`, then `cargo_expand` needs to import `config`. Preference?